packages/accordions/src/styled/StyledIcon.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import {
+  getColor,
+  getLineHeight,
+  retrieveComponentStyles,
+  DEFAULT_THEME
+} from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'accordions.step_icon';
+
+interface IStyledIcon {
+  isActive?: boolean;
+  isHorizontal?: boolean;
+}
+
+export const StyledIconFlexContainer = styled.div`
+  display: flex;
+  flex-basis: 100%;
+  justify-content: center;
+  width: 100%;
+`;
+
+const sizeStyles = (props: IStyledIcon & ThemeProps<DefaultTheme>) => {
+  const size = `${props.theme.space.base * 6}px`;
+
+  return css`
+    margin-bottom: ${props.isHorizontal && `${props.theme.space.base * 2}px`};
+    /* stylelint-disable-next-line property-no-unknown */
+    margin-${props.theme.rtl ? 'left' : 'right'}: ${!props.isHorizontal &&
+    `${props.theme.space.base * 3}px`};
+    width: ${size};
+    min-width: ${size};
+    height: ${size};
+    min-height: ${size};
+    line-height: ${getLineHeight(size, props.theme.fontSizes.sm)};
+    font-size: ${props.theme.fontSizes.sm};
+  `;
+};
+
+const colorStyles = (props: IStyledIcon & ThemeProps<DefaultTheme>) => {
+  return css`
+    background: ${props.isActive
+      ? getColor('neutralHue', 600, props.theme)
+      : getColor('neutralHue', 200, props.theme)};
+    color: ${props.isActive
+      ? props.theme.colors.background
+      : getColor('neutralHue', 800, props.theme)};
+  `;
+};
+
+export const StyledIcon = styled.div.attrs<IStyledIcon>({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledIcon>`
+  display: flex;
+  align-items: center;
+  align-self: ${props => !props.isHorizontal && 'self-start'};
+  justify-content: center;
+  transition: background 0.25s ease-in-out, color 0.25s ease-in-out;
+  border-radius: 100%;
+  ${colorStyles}
+  ${sizeStyles}
```

Comment:
Switch these two. `colorStyles` always take priority over `sizeStyles`. See https://github.com/zendeskgarden/react-components/blob/master/docs/api.md#rules for details.